UnityProject/Assets/Scripts/UI/ControlChat.cs

```diff
@@ -126,10 +129,12 @@ private void Update()
 
 		if (UIManager.IsInputFocus)
```

Comment:
You can reduce the if statement nesting a little by combining these
```suggestion
		if (UIManager.IsInputFocus && KeyboardInputManager.IsEnterPressed())
```